### PR TITLE
Fix remote nested library fetch and add test

### PR DESCRIPTION
**Pull request by KwisatzHaderach**
Fix for libraries that are remote and in nested fmf tree.
Accompanying patch in tests repo https://github.com/teemtee/tests/pull/5.

Fixes https://github.com/teemtee/tmt/issues/2149

Pull Request Checklist

* [x] implement the feature
* [x] extend the test coverage

### CODE REVIEW

File: tests/libraries/nested/data/test.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "pushd $tmp"
+        rlRun "rlImport tests/nested/file"
```
Comment (psss) [inline tests/libraries/nested/data/test.sh:8]: Just want to make sure: Shall the `rlImport` command be given the repository name as well? Shouldn't it be just `rlImport nested/file`. @sopos, any thoughts on this?

File: tests/libraries/nested/test.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup "Prepare test"
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest
+        rlRun "set -o pipefail"
+        rlRun "tmt run -ar discover -vvvddd report -vvv 2>&1 >/dev/null | tee output"
```
Comment (psss) [inline tests/libraries/nested/test.sh:11]:
```suggestion
        rlRun -s "tmt run -ar discover -vvvddd report -vvv 2>&1 >/dev/null"
```

What about using `rlRun -s` to capture the output and dropping the `pipefail` above?

**Review by psss — COMMENTED** [review]
Thanks for fixing this! Looks good, just two questions.

File: tests/libraries/nested/data/test.sh (hunk 1 above)
Comment (KwisatzHaderach) [reply tests/libraries/nested/data/test.sh:8]: Yeah, repository name was always part of the rlImport path. As discussed privately.

Comment (psss) [reply tests/libraries/nested/data/test.sh:8]: Library naming confirmed to be ok on the hacking session as well.

**Review by psss — APPROVED** [review]

**Review by happz — APPROVED** [review]

**Pull request merged**